packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowCheck.tsx

```diff
@@ -62,6 +62,7 @@ const DetailsRowCheckBase = (props: IDetailsRowCheckProps) => {
       className={css(classNames.root, classNames.check, checkClassNames.checkHost)}
       aria-checked={isPressed}
       data-selection-toggle={true}
+      data-is-focusable={true}
```

Comment:
This doesn't seem right. the DIV is not focusable.

Reply:
When the entire row is focused, you can press space to toggle the details row on and off. There is no need to make the check icon focusable.

Reply:
Thanks, for letting me know. i din't knew that when we press space it would work as toggle. i will close this pull request.